documentation/ci-fix.md

```diff
@@ -1,17 +1,47 @@
+# Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [CI Fix Guide](#ci-fix-guide)
+  - [Prerequisites](#prerequisites)
+  - [`Swagger SpellCheck`](#swagger-spellcheck)
+  - [`Swagger PrettierCheck`](#swagger-prettiercheck)
+    - [Prettier reference](#prettier-reference)
+  - [`Swagger ModelValidation`](#swagger-modelvalidation)
+  - [`Swagger SemanticValidation`](#swagger-semanticvalidation)
+  - [`Swagger BreakingChange` and `BreakingChange(Cross-Version)`](#swagger-breakingchange-and-breakingchangecross-version)
+    - [Adding label on PR automatically](#adding-label-on-pr-automatically)
+    - [Run `oad` locally](#run-oad-locally)
+  - [`Swagger LintDiff` and `Swagger Lint(RPaaS)`](#swagger-lintdiff-and-swagger-lintrpaas)
+  - [`Swagger LintDiff` for TypeSpec: troubleshooting guides](#swagger-lintdiff-for-typespec-troubleshooting-guides)
+    - [`Record<unkown>` causes `AvoidAdditionalProperties` and `PropertiesTypeObjectNoDefinition`](#recordunkown-causes-avoidadditionalproperties-and-propertiestypeobjectnodefinition)
+    - [`RequestBodyMustExistForPutPatch`](#requestbodymustexistforputpatch)
+    - [`PatchPropertiesCorrespondToPutProperties`](#patchpropertiescorrespondtoputproperties)
+  - [`Swagger Avocado`](#swagger-avocado)
+    - [Get help fixing Avocado validation failures](#get-help-fixing-avocado-validation-failures)
+    - [Run avocado locally](#run-avocado-locally)
+  - [`Swagger ApiDocPreview`](#swagger-apidocpreview)
+  - [`TypeSpec Validation`](#typespec-validation)
+    - [Run `tsv` locally](#run-tsv-locally)
+  - [Suppression Process](#suppression-process)
+  - [Checks not covered by this guide](#checks-not-covered-by-this-guide)
+  - [Obsolete checks](#obsolete-checks)
+
 # CI Fix Guide
 
 Short link: https://aka.ms/azsdk/ci-fix
 
 This page provides detailed instructions on how to diagnose, reproduce, fix and get help on various [automated validation tooling] failures on your [Azure REST API specs PR].
 
+For more help, see [aka.ms/azsdk/pr-getting-help] and [aka.ms/azsdk/support].
+
 ## Prerequisites
 
 Most guides here require for you to have `npm` installed, which you can get by installing [Node.js](https://nodejs.org/en/download).
 
-## Spell check
+## `Swagger SpellCheck`
```

Comment:
OpenAPI?

Reply:
Matching checks name for now, on purpose.